docs/team/TSAI-HSIAO-HAN.adoc

```diff
@@ -88,23 +88,23 @@ Improved current `delete` command so that when a bookmark been deleted it's remi
 This part is about the sections I contributed to the User Guide.
 The section extracted from the User Guide below showcase my ability to write a formal document to our target users.
 
-==== _------------- This is the start of extraction -------------_
+_------------- This is the start of extraction -------------_
 include::../UserGuide.adoc[tag=ReminderCommand]
 include::../UserGuide.adoc[tag=sort]
 
-==== _------------- This is the end of extraction -------------_
+_------------- This is the end of extraction -------------_
 
 == Contributions to the Developer Guide
 This part is about the sections I contributed to the Developer Guide.
 The section extracted from the Developer Guide below showcase my ability to write a technical documentation to a developer and show technique depth of my contribution.
 
-==== _------------- This is the start of extraction -------------_
+_------------- This is the start of extraction -------------_
```

Comment:
```suggestion
_------------- This is the start of the excerpt -------------_
```

Reply:
<img width="1046" alt="Screenshot 2019-11-11 at 1 24 17 PM" src="https://user-images.githubusercontent.com/46101221/68562945-c7a79980-0486-11ea-8db2-59f3e17b6119.png">

This part is showing up weirdly on the Netlify preview.